common/src/main/java/com/wynntils/mc/mixin/ClientLevelMixin.java

```diff
@@ -6,13 +6,19 @@
 
 import com.wynntils.mc.EventFactory;
 import net.minecraft.client.multiplayer.ClientLevel;
+import net.minecraft.client.player.AbstractClientPlayer;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 @Mixin(ClientLevel.class)
 public abstract class ClientLevelMixin {
+    @Inject(method = "addPlayer(ILnet/minecraft/client/player/AbstractClientPlayer;)V", at = @At("HEAD"))
+    private void addPlayer(int id, AbstractClientPlayer player, CallbackInfo ci) {
+        EventFactory.onPlayerJoinedWorld(player);
```

Comment:
To be sure, this is the change so we load our own user's info too?

Reply:
It is, yes. The method we used to mix into delegated to this one, actually - but it was the packet listener class, so it was only handling other players